benchmarks.toml

```diff
@@ -1,65 +1,129 @@
+[build]
+[build.nvidia]
```

Comment:
(not for this PR) @johnbowen42 can we have build variants too? (e.g., build.nvidia.aot)